app/GssProxy_Daemon/doc/README.head.md

```diff
@@ -0,0 +1,8 @@
+Basic monitoring of the [GssProxy Daemon](https://pagure.io/gssproxy).
+
+## Usage
+1. Import the
```

Comment:
It looks like these usage instructions could move to the template as they are all over the place.